src/prompts/alert/alert.scss

```diff
@@ -0,0 +1,109 @@
+.kumulos-alert-container {
+    background: #fff;
+
+    position: fixed;
+
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    max-height: 100%;
+    padding: 20px;
+
+    display: flex;
+    flex-direction: column;
+
+    font-family: $defaultFont;
+
+    animation: kumulos-anim-fade-in 0.2s ease-in;
+    backface-visibility: hidden;
+}
+
+@media only screen and (min-device-width: 480px){
+  .kumulos-alert-container {
+    border-radius: 0 0 3px 3px;
+    border-color: #fff;
+
+    left: 50%;
+
+    width: 500px;
+    height: auto;
+    min-height: 250px;
+
+    margin-left: -250px;
+
+    box-shadow: 0 0 3px 1px $shadow-color;
+  }
+}
+
+
+.kumulos-alert-header {
+  display: flex;
+  font-size: 24px;
+
+  h1 {
+    font-size: inherit;
+    flex: 1;
+    margin: 0;
+  }
+
+  .kumulos-alert-icon {
+    width: 70px;
+    height: 70px;
+
+    img {
+        width: 100%;
+    }
+  }
+}
+
+.kumulos-alert-body {
+  flex: 1;
+  white-space: pre-wrap;
+  overflow-y: auto;
+}
+
+.kumulos-alert-actions {
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 20px;
+}
+
+
+.kumulos-alert-action-button {
+  border: none;
+
+  margin-left: 10px;
+  padding: 7px 10px;
+
+  min-width: 100px;
+
+  text-transform: uppercase;
+  font-size: 16px;
+  font-weight: 600;
+
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.kumulos-alert-action-button-cancel {
+    background: none;
+    color: #555;
+
+    &:hover {
+      background: #eee;
+    }
+}
+
+.kumulos-alert-action-button-confirm {
```

Comment:
Thinking ahead a little seeing as we're likely to pick up some of the additional modals/prompt styles, should we be thinking more generically about buttons as a util to reuse, like kumulos-action-button etc.? Ofc we could always just keep the -alert- stuff and use in other components.

Reply:
Yeah, just reflecting on this -  good idea just to rename now - I'll just drop the `alert` from the classes and move these into a buttons partial.

Reply:
Done 👍